forestserviceprototype/specialuseform/templates/specialuseform/applications.html

```diff
@@ -0,0 +1,29 @@
+{% extends "specialuseform/base.html" %}
+
+{% block content %}
+  <h1>Applications</h1>
+  {% if permits %}
```

Comment:
Should this have an `{% else %}` down the line, so that there's some content even if there are no permits?